embassy-usb/src/builder.rs

```diff
@@ -15,6 +22,11 @@ pub struct Config<'a> {
     pub(crate) vendor_id: u16,
     pub(crate) product_id: u16,
 
+    /// Device BCD USB version.
+    ///
+    /// Default: `0x0210` ("2.1")
+    pub bcd_usb: BcdUsbVersion,
```

Comment:
I'd name this just `UsbVersion`. 

"BCD" is a reference to Binary Coded Decimal, ie how it's coded in binary. if it's an enum the user no longer cares about that.